smoke/Makefile

```diff
@@ -1,6 +1,6 @@
 PACKAGES ?= $(shell go list ./... | grep -v /vendor/)
 GOPROXY ?= https://goproxy.io
-TESTS ?= .*
+TESTS ?= 'TestAPI|TestBlobCache|TestCompatibility|TestImage|TestNativeLayer|TestZranLayer'
```

Comment:
Can we keep it with `.*`? Other developers to add test cases will forget to update this.

Reply:
If we keep `.*`, when we use `TESTS` in `make test` we will run `performance_test` together. However, the performance test needs some env setup.

Reply:
Maybe I can move the performance test to a separate directory in `smoke/tests` ?